backend/src/main/java/de/ricardo/backend/DiaryService.java

```diff
@@ -11,19 +11,21 @@
 public class DiaryService {
 
     private final DiaryRepository diaryRepository;
+    private final IdService idService;
 
-    DiaryService(DiaryRepository diaryRepository){
+    DiaryService(DiaryRepository diaryRepository, IdService idService){
```

Comment:
you could do this with an AllArgsConstructor annotation, but this works as well. 👍